gradle/libs.versions.toml

```diff
@@ -9,10 +9,11 @@ jspecify = "0.3.0"
 jdt-annotation = "2.2.700"
 android-annotation = "1.7.0"
 spotbugs-annotations = "4.7.3"
-openapi-generator = "6.6.0"
+openapi-generator = "7.0.0"
 swagger-parser = "1.0.67"
 swagger-parser-v3 = "2.1.16"
 javaparser = "3.25.5"
+commons-codec = "1.16.0"
```

Comment:
`commons-codec` is not necessary or used is it?

Reply:
@sdelamo It is used by openapi-generator, not remembered directly or through swagger-parser. In any case, the old version has a vulnerability. This fix will help remove this vulnerability from micronaut-openapi

Reply:
do you know about this? @melix

Reply:
@melix ping